pkg/cluster/deploystorage.go

```diff
@@ -219,18 +217,30 @@ func (m *manager) ensureGraph(ctx context.Context, installConfig *installconfig.
 	for _, a := range targets.Cluster {
 		err = g.Resolve(a)
 		if err != nil {
-			return err
+			return nil, err
 		}
 	}
 
 	// Handle MTU3900 feature flag
 	if m.doc.OpenShiftCluster.Properties.NetworkProfile.MTUSize == api.MTU3900 {
 		m.log.Printf("applying feature flag %s", api.FeatureFlagMTU3900)
 		if err = m.overrideEthernetMTU(g); err != nil {
-			return err
+			return nil, err
 		}
 	}
 
+	return g, nil
+}
+
+func (m *manager) ensureGraph(ctx context.Context, g graph.Graph) error {
```

Comment:
Now that this function only saves the graph, IMO it would be clearer to rename this function to `saveGraph` or `persistGraph`.

Reply:
ack